frontend/packages/insights-plugin/__tests__/insightswidget.spec.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import { mount } from 'enzyme';
```

Comment:
Try to use `shallow` for all 3 sets of tests instead of `mount`. The tests you've written should work the same if we stick to `shallow`. Try it out